indicators/models.py

```diff
@@ -215,6 +215,9 @@ class DisaggregationValue(models.Model):
     disaggregation_label = models.ForeignKey(DisaggregationLabel, on_delete=models.deletion.PROTECT)
     indicator = models.ForeignKey('indicators.Indicator', null=True, blank=True)
     value = models.CharField(max_length=765, blank=True)
+    table_uuid = models.CharField(max_length=765, blank=True)
+    column_name = models.CharField(max_length=765, blank=True)
```

Comment:
I don't have anything against, making it big. But are you going to have such a big column name?

Reply:
Updated.